itou/templates/dashboard/dashboard.html

```diff
@@ -421,8 +421,10 @@ <h5 class="card-header">Admin</h5>
                     <div class="card-body">
                         <ul class="list-unstyled">
                             <li class="card-text mb-3">
-                                <i class="ri-pulse-line ri-lg mr-1"></i>
+                                <i class="ri-file-copy-2-line ri-lg mr-1"></i>
                                 <a href="{% url 'siae_evaluations_views:siae_job_applications_list' %}">Justifier mes auto-prescriptions</a>
+                                {# static icon for openning feature, need to be hidden after proofs submission.#}
```

Comment:
nit: utilsier NOTE(vporte) ça se greppe plus facilement.
et "opening" sans deux N, needs.
Je ne suis pas sur de comprendre le commentaire en réalité :)

Reply:
mise à jour : `{# note vincentporte: static icon, will be displayed dynamically later.#}`